client/src/ContinueIcon.js

```diff
@@ -0,0 +1,106 @@
+const ContinueIcon = ({ color }) => {
```

Comment:
Nice, how did you get the color prop working?

Reply:
I had a simple syntax error. I wasn't using the curly braces, and realized I needed to destructure color. Then it worked!